zkml/src/layers/requant.rs

```diff
@@ -404,18 +404,35 @@ impl Requant {
             point: point.clone(),
             eval: first_claim.eval - E::from((*quantization::RANGE / 2) as u64),
         };
-        // println!("correct claim eval: {:?}", corrected_claim.eval);
-        // println!(
-        //    "output eval: {:?}",
-        //    output.to_vec().into_mle().evaluate(&corrected_claim.point)
-        //);
+
         // Add the claim used in the activation function
         same_poly_prover.add_claim(corrected_claim)?;
         let claim_acc_proof = same_poly_prover.prove(&same_poly_ctx, prover.transcript)?;
 
+        // Now we add the constant factor back here so that it is a claim for the committed lookup poly
+        let Claim {
+            point: acc_point,
+            eval: acc_eval,
+        } = claim_acc_proof.extract_claim();
+        let first_poly_claim = Claim::<E>::new(
+            acc_point,
+            acc_eval + E::from((*quantization::RANGE / 2) as u64),
+        );
+
         prover
             .witness_prover
-            .add_claim(requant_info.poly_id, claim_acc_proof.extract_claim())?;
+            .add_claim(requant_info.poly_id * 100, first_poly_claim)?;
+
+        logup_proof
+            .output_claims()
+            .iter()
+            .enumerate()
+            .skip(1)
```

Comment:
Could you explain why the skip happens via a small comments ? (in general more comments would be super welcome here !)